schemas/2015-06-01/Microsoft.KeyVault.json

```diff
@@ -0,0 +1,209 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.KeyVault.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.KeyVault",
+    "description": "Microsoft KeyVault Resource Types",
+    "definitions": {
+        "keyPermission": {
+            "anyOf": [
```

Comment:
I would suggest to define schema according what provider accepts as valid inputs.
If you really want to go with the PowerShell way, you need to define "all" and other enum values as two objects and use "oneOf" to refernce them in Ln57.